ouroboros-consensus/src/Ouroboros/Consensus/Protocol/Abstract.hs

```diff
@@ -215,6 +219,9 @@ class ( Show (ChainState    p)
 newtype SecurityParam = SecurityParam { maxRollbacks :: Word64 }
   deriving (Show, Eq, Generic, NoUnexpectedThunks)
 
+newtype NetworkMagic = NetworkMagic {unNetworkMagic :: Word32 }
```

Comment:
Missing space after the `{`

Reply:
Perhaps this type should go in the network lib and we can use it in `NodeToNodeVersionData` etc, without having to `unNetworkMagic` it.

Reply:
Where do you want to define the type? I don't want one NetworkMagic type for NodeToClient and one NetworkMagic type for NodeToNode.

Reply:
I've moved `NetworkMagic` to a new module, `Ourboros.Network.Magic`.